web/views/package_view.ex

```diff
@@ -2,11 +2,9 @@ defmodule HexWeb.PackageView do
   use HexWeb.Web, :view
 
   def show_sort_info(nil), do: "(Sorted by name)"
-  def show_sort_info(param) do
-    available_params = %{
-      "name" => "name", "inserted_at" => "recently created",
-      "downloads" => "downloads", "updated_at" => "recently updated"}
-
-    if Map.has_key?(available_params, param), do: "(Sorted by #{available_params[param]})"
-  end
+  def show_sort_info("name"), do: "(Sorted by name)"
+  def show_sort_info("inserted_at"), do: "(Sorted by recently created)"
+  def show_sort_info("updated_at"), do: "(Sorted by recently updated)"
+  def show_sort_info("downloads"), do: "(Sorted by downloads)"
+  def show_sort_info(param), do: nil
```

Comment:
We need to do this: "def show_sort_info(param), do: nil" for cases when user try sorting by the unavilable param. Without this we have an error.

And "def show_sort_info(nil), do: "(Sorted by name)"" for cases when the sorting aren't selected